octrace/UI/Controllers/MapViewController + MapFeature.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  MapFeature.swift
+//  octrace DEV
+//
+//  Created by Andrius Shiaulis on 01.05.2020.
+//  Copyright © 2020 OpenCovidTrace. All rights reserved.
+//
+
```

Comment:
Plz remove comments, we don't have them in the source code.

Reply:
done